core/src/main/java/com/google/instrumentation/stats/MutableDistribution.java

```diff
@@ -95,6 +96,16 @@ long getCount() {
   }
 
   /**
+   * The sum of the squares of values in the population. If {@link #getCount()} is zero then this
+   * value will also be zero.
+   *
+   * @return The sum of squares of values in the population.
+   */
+  double getSumSquares() {
```

Comment:
Ack: I've actually just decided to remove this from this PR, since I realized it can't actually be used the way I want. I'll do a separate PR for variance-related stuff.